google-cloud-spanner/src/main/java/com/google/cloud/spanner/PartitionedDmlTransaction.java

```diff
@@ -141,6 +141,10 @@ public void invalidate() {
   @Override
   public void setSpan(Span span) {}
 
+  // No-op method needed to implement SessionTransaction interface.
```

Comment:
nit: Either put this comment inside the method body, or leave it here but change it to Javadoc